src/browser/index.jsx

```diff
@@ -395,6 +401,12 @@ var MattermostView = React.createClass({
       }
     });
 
+    webview.addEventListener('page-title-updated', function(event) {
```

Comment:
Yes. So the windows title should be updated only for the current tab. If the title became one of non-active tab, it would be confusing. As I wrote at first, the listener would be able to know the state by using a prop.